index.html

```diff
@@ -2905,6 +2915,152 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls one or more
+particular confirmation methods and to bind the <a>claims</a> about the
+<a>subject</a> to one or more of these confirmation methods. In this way, an
+<a>issuer</a> explicitly enables a <a>verifier</a> to validate that the
+<a>holder</a> presenting the <a>verifiable credential</a> has proven control of
+one or more of these confirmation methods when the <a>claims</a> bound to the
+confirmation method are presented.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method or to use a different
+mechanism to validate the <a>holder</a> is bound to the presented <a>claims</a>
+if required without impacting liability if not specified by other means such
+as a <code>termsOfUse</code> policy.
+        </p>
+
+        <p>
+This specification defines the <code>confirmationMethod</code> <a>property</a>
+for expressing confirmation method information in a
+<code>credentialSubject</code> in a <a>verifiable credential</a>.
+        </p>
+
+        <p class="note">
+For example, an <a>issuer</a> can include a confirmation method based on public
+key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
+generate and include a <a>proof</a> with a cryptographic signature in the
+<a>verifiable presentation</a> where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>. A <a>verifier</a> can validate that the <a>holder</a> controls
+the confirmation method by verifying the <a>proof</a> of the <a>verifiable
+presentation</a> using the information in the confirmation method. The
+confirmation method can include the verification key or the type of the
+confirmation method can define that the verification key is inferred by other
+<a>properties</a> in the <a>verifiable credential</a> such as the
+<code>credentialSubject.id</code>.
+        </p>
+
+        <dl>
+          <dt><dfn>confirmationMethod</dfn></dt>
+          <dd>
+If present, the value of the <code>confirmationMethod</code> <a>property</a> is
```

Comment:
as stated in the F2F I'm opposed to this text _without_ a requirement on the Issuer to verify that the holder actually has access to these confirmation method(s). Otherwise we're just creating more opportunity for failure.

Reply:
That makes sense, I will add such a requirement to the text.

Reply:
@decentralgabe I added some text, can you please check if this is fine with you now?